src/steamship/agents/schema/action.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any, List, Optional, Union
+
+from pydantic import BaseModel
+
+from steamship import Block, Task
+from steamship.agents.schema.tool import AgentContext, Tool
+
+
+class Action(BaseModel):
```

Comment:
Not important now, but thought pin to wonder if eventually `input` and `output` will benefit from being a proxy object which represents: "The ability to load blocks -- now or in the future".  Such that both bindings can be a persistence-friendly future on blocks.